test/suite/helper.py

```diff
@@ -96,6 +96,29 @@ def confirm_empty(self, uri):
         self.assertEqual(cursor.next(), wiredtiger.WT_NOTFOUND)
     cursor.close()
 
+# copy a WT home directory
+def copy_live_wiredtiger_home(olddir, newdir, aligned=True):
+    # unaligned copy requires 'dd', which may not be available on Windows
+    if not aligned and os.name == "nt":
+        raise AssertionError(
+            'copy_live_wiredtiger_home: unaligned copy impossible on Windows')
+    shutil.rmtree(newdir, ignore_errors=True)
+    os.mkdir(newdir)
+    for fname in os.listdir(olddir):
+        fullname = os.path.join(olddir, fname)
+        # Skip lock file, on Windows it is locked.
+        if os.path.isfile(fullname) and "WiredTiger.lock" not in fullname:
```

Comment:
One difference between the original and the new code is that the original did shutil.copytree on non-windows systems to simulate the expected `cp -rp` that a user would do.  The new code now copies files individually and skips the `WiredTiger.lock` file.  So we're no longer testing what a user is likely to do.  @michaelcahill is that difference important?